scripts/isct_convert_binary_to_trilinear.py

```diff
@@ -61,8 +61,10 @@ def main():
             opts, args = getopt.getopt(sys.argv[1:], 'hi:v:r:s:')
         except getopt.GetoptError:
             usage()
+            raise SystemExit(2)
         if not opts:
             usage()
+            raise SystemExit(2)
         for opt, arg in opts:
             if opt == '-h':
                 usage()
```

Comment:
I think we need a `raise SystemExit(0)` here or else `usage()` gets called twice.

https://github.com/neuropoly/spinalcordtoolbox/blob/9a928bf254ade9a4933605e74a66225262b32169/scripts/isct_convert_binary_to_trilinear.py#L78-L80

(I also think another `raise SystemExit(2)` is needed after this `fname_data` `usage()` call.)

Reply:
What about just using `return` instead of `raise SystemExit(0)`?

Reply:
Oh no of course you were right.